rpm/fingertip.spec

```diff
@@ -0,0 +1,83 @@
+%global git_date 20200401
+%global git_commit c5a3bb6dff803b4d9c51aa94d3982fb5a0003f88
+%{?git_commit:%global git_commit_hash %(c=%{git_commit}; echo ${c:0:7})}
+%global rpm_release 1
+
+Name:		fingertip
+Version:	0.%{git_date}
+Release:	%{rpm_release}.git%{git_commit_hash}%{?dist}
+Summary:	Control VMs, containers and other machines with Python, leverage live snapshots
+
+License:	GPLv3+
+URL:		https://github.com/t184256/fingertip
+Source0:	https://github.com/t184256/fingertip/archive/%{git_commit}/%{name}-%{git_commit_hash}.tar.gz
+Source1:	fingertip.in
+
+BuildArch:	noarch
+BuildRequires:	python3
+BuildRequires:	python3-devel
+
+Requires:	podman
```

Comment:
Should be a soft-dependency, I think, given it's a PoC, should be unpopular and has to be specified explicity.

Reply:
Right. The containerized version is not packaged here, so this should not be here. I will remove.

Reply:
I meant, there's `fingertip/plugins/backend/podman-criu.py`, but can't invoke it without having `podman` mentioned in the command, and that's enough of a hint that you need podman.

Reply:
Oh. My bad reading. Returned as Recommends now.